var/spack/repos/builtin/packages/py-pytouchreader/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPytouchreader(PythonPackage):
+    """Python interface to interact with touch files."""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/PyModules"
+    url      = "ssh://bbpcode.epfl.ch/hpc/PyModules"
+
+    version('develop', git=url, clean=False)
```

Comment:
only develop? no re-producible modules then :)

Reply:
Need to start tagging the repo. I've got some changes coming up, too (better diff to aid with TD development)